model/aws.py

```diff
@@ -28,5 +28,11 @@ def access_key_id(self):
     def secret_access_key(self):
         return self.raw['secret_access_key']
 
+    def rotation_cfg_name(self):
+        return self.raw.get('rotation_cfg_name')
+
+    def iam_user_name(self):
```

Comment:
considering presense of `rotation_cfg_name` implies `iam_user_name` must also be set, how about adding this into validation. there are two options:

1. calculate required attributes in `_required_attributes` method
2. validate in `validate` method (see e.g. model/concourse.py for an example)

Reply:
Will add to required attributes, however we must ensure to explicitly validate in `aws.py`, as secret-replication handles validation-errors with warnings.